src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1682,7 +1688,27 @@ static bool processSubscriptions
     }
 
     /* Check 2: expression (q) */
-    if (!matchExpression(notifyCerP, trigs->expression.q))
+    if (noCache == true)
```

Comment:
Sure, but ... it will complicate things a little.
Creating it just-in-time and then removing it is also not a bad idea.
Unless it has to be created more than once, of course. That would be a mistake.